docs/fsharp/language-reference/byrefs.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Byrefs (F#)
+description: Learn about byref and byref-like types in F#, which are used for low-level programming.
+ms.date: 09/02/2018
+---
+
+# Byrefs
+
+F# has two major concepts that deal in the space of low-level programming:
```

Comment:
These aren't concepts; they're features. And the implication of this statement is that this is the totality of low-level programming concepts in F#. Is that true?

Reply:
Not the totality, but certainly a big deal. I'll update.